### ISSUE TITLE
Side navigation on SAVi pages

### ISSUE THREAD

**Issue by robredpath** [IATI/D-Portal#679]
Activity pages can be quite long, especially if there is a lot of information to include. [This](https://d-portal.org/ctrack.html?reporting_ref=XM-OCHA-HPC8568#view=act&aid=XM-OCHA-HPC8568-XM-DAC-7-PPR-4000002150) is a bit of an extreme example, but illustrates the point - lots of expenditure lines, lots of indicator lines. 

It would be helpful to have some kind of page navigation to help people understand where they are on the page - what's above, what's below, and where they are. And, of course, to use that to skip to an area of interest rather than scrolling.

**Comment by notshi** [comment]
Agreed - we used to have collapsible headers for each section but a navigation of sorts sounds doable.

**Comment by notshi** [comment]
This is a longer term project and should not be included in the design system (and docs) milestone - will remove.
